tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -216,6 +241,30 @@ describe('main_html5 wrapper tests', function () {
     expect(vtc.notifyParameters).to.eql([vtc.interface.EVENTS.SEEKED]);
   });
 
+  it('should not raise seeked when initial time is set to non-zero', function(){
```

Comment:
It looks like previous test contains this one and then does more stuff, so is this test required?